snails-backend-everything.el

```diff
@@ -83,7 +86,25 @@
  (lambda (input)
    (when (and (executable-find "es")
               (> (length input) 3))
-     (list "es" "-n" "30" input)))
+     (let ((search-dir nil)
```

Comment:
Use search-dir instead `(search-dir nil)` , because nothing to set, let variable default is nil.

Reply:
thanks for your review.
done.